src/app/components/share/share.scss

```diff
@@ -6,31 +6,32 @@ $linkedin-blue: #0077b5;
 
 .share-buttons {
     .buttons {
-        display: flex;
+        display: grid;
+        grid-gap: 1rem;
         justify-content: center;
-        margin: 1rem 0;
-
-        &:not(.minimal) {
```

Comment:
The `minimal` option is the only way it was being called, so I got rid of the non-minimal configuration